src/App/MessagePageClass.test.tsx

```diff
@@ -44,7 +44,7 @@ test('MessagePage link is hidden', () => {
   ).find(MessagePage);
   msg.setState({ linkHidden: true, startTime: 0 });
   (msg.instance() as MessagePage)._animate(125)();
-  expect(msg.html()).not.to.contain('Go Home');
+  expect(msg.html()).not.to.contain('messagePage.goHome');
```

Comment:
why isnt it coming in english if it is the default language?

Reply:
Because there's no translations loaded.